### DISCUSSION TITLE
Mis-match in nbr endpoints in model & in data

### DISCUSSION THREAD

**Discussion by filippo-ciceri** [nlmixrdevelopment/nlmixr#574] [Q&A] [answered]
Hello nlmixr development team,

Could you please provide a little bit of context around this error message?

Thank you very much.
Filippo

**Comment by mattfidler** [comment, marked as answer]
Hi Filippo,

I think it means the data is setup incorrectly for a multiple endpoint model.  I would check your data to see if it is setup correctly.

Matt

**Reply by filippo-ciceri** [reply]
Hello Matthew,

thank you for the quick reply. I am afraid this is not possible. 
If you could give me some suggestions on what to check or a link on how to format the data, that would be incredibly helpfull.

Thank you
Filippo

**Reply by mattfidler** [reply]
Sure 
- https://nlmixrdevelopment.github.io/RxODE/articles/RxODE-datasets.html
- https://nlmixrdevelopment.github.io/RxODE/articles/RxODE-event-types.html

**Reply by filippo-ciceri** [reply]
Perfect. Thank you.
